cert_manager/manifest.json

```
@@ -0,0 +1,30 @@
+{
+  "display_name": "cert-manager",
+  "maintainer": "ara.pulido@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "cert_manager",
+  "metric_prefix": "cert_manager.",
+  "metric_to_check": "cert_manager.prometheus.health",
+  "creates_events": false,
+  "short_description": "Track all your cert-manager metrics with Datadog",
+  "guid": "c9bdaf11-fe15-4892-ae30-47c5124144e5",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-cert_manager Integration",
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "cert-manager",
```

…And should this be `cert_manager` (the name of the folder)?

```suggestion
  "integration_id": "cert_manager",
```

---

I didn't do this one because if I do, I get a validation issue:
```
ddev validate manifest
Validating all manifest.json files...
cert_manager/manifest.json... FAILED
integration_id contains invalid characters
```

---

Ah, indeed… We enforce alphanumerical characters and hyphens here. :-)